multiple definition of `gen9_hevc_encoder_kernels with gcc-10

There are multiple definition of `gen9_hevc_encoder_kernels' as experienced with gcc-10.
Since version 10, [gcc defaults to `-fno-common`](https://gcc.gnu.org/gcc-10/porting_to.html#common).

> /usr/bin/ld: ./.libs/libi965_drv_video.a(libi965_drv_video_la-gen9_hevc_encoder.o):/builddir/build/BUILD/intel-vaapi-driver-2.4.0/src/gen9_hevc_enc_kernels_binary.h:34: multiple definition of `gen9_hevc_encoder_kernels'; ./.libs/libi965_drv_video.a(libi965_drv_video_la-gen9_hevc_enc_kernels_binary.o):/builddir/build/BUILD/intel-vaapi-driver-2.4.0/src/gen9_hevc_enc_kernels_binary.h:34: first defined here
> collect2: error: ld returned 1 exit status

--- a/src/gen9_hevc_enc_kernels_binary.h
+++ b/src/gen9_hevc_enc_kernels_binary.h
@@ -31,6 +31,6 @@
 
 #define GEN9_HEVC_ENC_KERNEL_SIZE 149296
 
-const unsigned int gen9_hevc_encoder_kernels[GEN9_HEVC_ENC_KERNEL_SIZE];
+extern const unsigned int gen9_hevc_encoder_kernels[GEN9_HEVC_ENC_KERNEL_SIZE];
 
 #endif

--- a/test/test.h
+++ b/test/test.h
@@ -147,7 +147,7 @@ public:
             status = "VA_STATUS_ERROR_HW_BUSY"; break;
         case VA_STATUS_ERROR_UNSUPPORTED_MEMORY_TYPE:
             status = "VA_STATUS_ERROR_UNSUPPORTED_MEMORY_TYPE"; break;
-        case VA_STATUS_ERROR_UNKNOWN:
+        case VAStatus(VA_STATUS_ERROR_UNKNOWN):
             status = "VA_STATUS_ERROR_UNKNOWN"; break;
         default:
             status = "Unknown VAStatus";

